### PR TITLE
Some updates to tasks

### PR DESCRIPTION
**Pull request by joshuaGlass808**
old:
image

new:
image

- Fixed a bug related to creating tasks.

### CODE REVIEW

File: resources/js/Pages/Tasks/Task.vue [hunk 1]
```diff
@@ -40,9 +40,9 @@
 <template>
   <Layout>
     <div id='task'>
-      <h4 class='mb-5'>Task</h4>
+      <h4 class='mb-5'>Task #{{ task.id }}</h4>
       <section class='grid-col-3 mb-3'>
-        <p><b>Task ID:</b> {{ task.id }}</p>
+        <p><b>Title:</b> {{ task.title }}</p>
         <p><b>Author:</b> {{ task.author.name }}</p>
         <p><b>Created:</b> {{ new Date(task.created_at).toLocaleDateString() }}</p>
```
Comment (joshuaGlass808) [inline resources/js/Pages/Tasks/Task.vue:47]: @jlrusso do we plan on turning these into the ../shared/Input component later? That way they can be edittable?

File: resources/js/Pages/Tasks/Task.vue [hunk 2]
```diff
@@ -72,7 +72,6 @@
           :targetProps="['id','name']"
           heading='State'/>
       </section>
-      <p><b>Subject:</b> {{ task.title }}</p>
```
Comment (joshuaGlass808) [inline resources/js/Pages/Tasks/Task.vue:75]: We call it title in the creation part. I figured I would remove it and place it where we show the ID, as we can just nicely place that up by the word Task :)

File: resources/js/Pages/Tasks/Task.vue (hunk 1 above)
Comment (jlrusso) [reply resources/js/Pages/Tasks/Task.vue:47]: Yeah

File: resources/js/Pages/Tasks/Task.vue (hunk 2 above)
Comment (jlrusso) [reply resources/js/Pages/Tasks/Task.vue:75]: Sounds good

**Review by jlrusso — APPROVED** [review]

**Pull request merged**